resources/views/cart/item.blade.php

```diff
@@ -28,6 +28,18 @@ class="mx-auto"
                         </template>
                     </div>
                     @include('rapidez::cart.item.remove')
+                    <div v-if="item.backorder_count" class="flex gap-2">
+                        <x-heroicon-o-exclamation-circle class="mt-px w-5" />
+                        <span>
+                            <template v-if="item.backorder_count == item.qty">
```

Comment:
It looks like this if case is flipped, or could be `item.backorder_count < item.qty`

Reply:
I'd say the other way around, if the qty is the stock and it's lower than the backorder count, there are products to be backordered.